.editorconfig

```diff
@@ -1,26 +1,16 @@
 # EditorConfig is awesome: http://EditorConfig.org
 
-# Top-most EditorConfig file
 root = true
 
-# Unix-style newlines with a newline ending every file
 [*]
+charset = utf-8
 end_of_line = lf
 insert_final_newline = true
-
-# JS / PHP
-[*.{js,php,phpt}]
-charset = utf-8
-indent_style = tab
-indent_size = 4
-
-# NEON
-[*.neon]
-charset = utf-8
+trim_trailing_whitespace = true
 indent_style = tab
-indent_size = 4
+indent_size = tab
```

Comment:
Make exception for markdown, it cannot be tabbed (due to Github red blackground)